lib/code_corps/github/sync/comment/comment/changeset.ex

```diff
@@ -22,22 +22,22 @@ defmodule CodeCorps.GitHub.Sync.Comment.Comment.Changeset do
   @spec build_changeset(Comment.t, map, GithubComment.t, Task.t, User.t) :: Changeset.t
   def build_changeset(
     %Comment{id: nil} = comment,
-    %{"comment" => attrs},
+    attrs,
```

Comment:
`%{} = attrs` would provide some type checking here. Not strictly needed, but you mentioned before you prefer to have it.